proposals/lip-0042.md

```diff
@@ -108,7 +100,7 @@ getBlockReward(blockHeader):
   return defaultReward
 ```
 
-Here, `/` represents integer division, `isValidSeedReveal` is the function exposed by the [random module][research:random-module] and `impliesMaximalPrevotes` is the function exposed by the [BFT module][research:bft-module#specs].
+Here `isValidSeedReveal` is the function exposed by the [random module][research:random-module] and `impliesMaximalPrevotes` is the function exposed by the [BFT module][research:bft-module#specs].
```

Comment:
Why did you remove the reamrk about the integer division? In python 3 `/` represents true (float) division. If you want to do floor division (or integer division) we should use `//`: https://stackoverflow.com/questions/19507808/python-3-integer-division

Reply:
Forgot this detail, changed back.